core/src/player.rs

```diff
@@ -488,7 +488,7 @@ impl Player {
                 let timer = Instant::now();
                 self.run_frame();
                 let elapsed = timer.elapsed().as_millis();
-                if frame > 0 && elapsed == 0 {
```

Comment:
Removing this check mean that if the first frame took 0ms then the next non catchup frame will start 1ms late(unless frame_accumulator is exactly frame_time).